### PR TITLE
provide tikz with degrees, not radians

### PR DESCRIPTION
**Pull request by dimpase**
as mentioned on https://github.com/networkx/networkx/issues/6357, tikz wants degrees (or radians, but these must be suffixed with `r`)

### CODE REVIEW

File: networkx/drawing/nx_latex.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def to_latex_raw(
         pos = nx.get_node_attributes(G, pos)
     if not pos:
         # circular layout with radius 1
-        pos = {n: f"({round(2* 3.1415 * i / len(G), 3)}:10)" for i, n in enumerate(G)}
+        pos = {n: f"({round(360.0 * i / len(G), 3)}:10)" for i, n in enumerate(G)}
```
[the remark this answers is not in the document]
Comment (dimpase) [reply networkx/drawing/nx_latex.py:221]: image

here is what I get with `default_radius=2`. Looks about right to me.

**Comment by dimpase** [comment]
Should there be a key argument `default_radius=` in this function?

**Comment by dschult** [comment]
Yay!  Thanks for this input and sleuthing. We need to change to degrees!

And we should find a way to easily make the default layout for the latex/tikz functions.
Perhaps a separate function could provide the `pos` argument... something like `tikz_circular_layout(G, radius)`  I'd prefer that over an argument to `to_latex` that influences the default layout. Let's just pick a good default and make it easy for people to make `pos` for tikz.  Thoughts?

**Comment by dimpase** [comment]
I'd say that layout is something which is more generally useful. Thus it's natural to introduce
`layout` method for graphs, like in [igraph](https://igraph.org/python/tutorial/0.9.6/visualisation.html#graph-layouts), [SageMath](https://doc.sagemath.org/html/en/reference/plotting/sage/graphs/graph_plot.html)

(careful with borrowing the code there - these two are under GPL, so it won't fly with networkx)

----

I propose to merge this PR as is, and work on the layout in another issue.

File: networkx/drawing/nx_latex.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (MridulS) [reply networkx/drawing/nx_latex.py:221]: Could you maybe change the default radius from 10 to 2 here?

**Comment by MridulS** [comment]
We have our set of layouts: https://networkx.org/documentation/latest/reference/drawing.html#module-networkx.drawing.layout, currently we just don't use them for latex graph drawing. TikZ provides with some layouts https://tikz.dev/gd-usage-tikz#sec-28.10, maybe we should investigate that further rather than using our inbuilt ones by default? But I agree we should do layouts in another PRs :)

**Comment by dschult** [comment]
So, shall we set the default to what it was before only this time do the radians correctly and set the radius to 1 (or maybe 2)?

The current default is not good, and I'd love for it to look reasonable.

**Comment by dimpase** [comment]
Radians are extra arithmetic, in this context - it's easier to divide 360 by $n$ than $2\pi$ by $n$. I'll change the default radius to 2 - although ideally it should depend upon $n$.

**Review by dschult — APPROVED** [review]

**Pull request merged**